rubin_sim/maf/batches/run_moving_join.py

```diff
@@ -8,12 +8,11 @@
 
```

Comment:
Curious - why did this one (run_moving_join.py) end up in rubin_sim/maf/batches, while the other similar scripts (run_moving_calc.py, for example) end up in rubin_sim/maf ?

Reply:
I think I was trying to place the method were they seem like they belonged.. but I am not sure I got all of them in the correct place..